.github/scripts/auto-label.cjs

```diff
@@ -0,0 +1,238 @@
+/**
+ * This Script can be tested locally with
+ *
+ * export GH_TOKEN=**** GITHUB_REPOSITORY=quisquous/cactbot PR_NUMBER=$NUM
+ * node ./.github/scripts/auto-label.cjs
+ */
+'use strict';
+const path = require('path');
+
+const json5 = require('json5');
+const lodash = require('lodash');
+const recast = require('recast');
+const github = require('@actions/github');
+const babelParser = require('recast/parsers/babel');
+const { HttpClient } = require('@actions/http-client');
+
+const prefixLabelMap = {
+  'resources/': 'resources',
+  'util/': 'util',
+  'test/': 'test',
+  '.mocharc.cjs': 'test',
+  'eslint/': 'style',
+  '.eslintrc.js': 'style',
+  '.github/workflows/': 'ci',
+  '.github/scripts/': 'ci',
+  'ui/config/': 'config',
+  'ui/dps/': 'dps',
+  'ui/eureka/': 'eureka',
+  'ui/fisher/': 'fishing🎣',
+  'ui/jobs/': 'jobs',
+  'ui/oopsyraidsy/': 'oopsy',
+  'ui/pullcounter/': 'pullcounter',
+  'ui/radar/': 'radar',
+  'ui/raidboss/': 'raidboss',
```

Comment:
Were you still planning to add a raidemulator label as we discussed on the other PR, or should that be a followup PR?

Reply:
I'm waiting for a conclusion or a decision. I think we didn't get one in previous PR.

Reply:
I think there weren't strong opinions.  I think separating sounds good.  In other words, if something is only a raidemulator change then it gets a raidemulator label and not a raidboss label.

Reply:
just done